polaris/management/commands/testnet.py

```diff
@@ -320,3 +417,162 @@ def get_balance(self, code, issuer_public_key, json) -> Optional[str]:
                 and balance_obj.get("asset_issuer") == issuer_public_key
             ):
                 return balance_obj["balance"]
+
+    def create_trustline(self, code :str, asset_issuer_public_key: str, distribution: Keypair):
+        accounts = self.get_or_create_accounts([distribution])
+        balance = self.get_balance(code, asset_issuer_public_key, accounts[distribution.public_key])
+        if balance:
+            print(f"\nTrustline to {code}:{asset_issuer_public_key} already exists")
+            return
+        print(f"\nCreating trustline to {code}:{asset_issuer_public_key}")
+        tb = TransactionBuilder(
+            self.server.load_account(distribution.public_key),
+            base_fee=settings.MAX_TRANSACTION_FEE_STROOPS
+            or self.server.fetch_base_fee(),
+            network_passphrase="Test SDF Network ; September 2015",
+        )
+        tb.append_change_trust_op(
+            asset=StellarSdkAsset(code=code, issuer=asset_issuer_public_key),
+            source=distribution.public_key,
+        )
+        envelope = tb.set_timeout(30).build()
+        envelope.sign(distribution)
+        try:
+            self.server.submit_transaction(envelope)
+        except BaseHorizonError as e:
+            print(
+                f"Failed to create trustline "
+                f"Result codes: {e.extras.get('result_codes')}"
+            )
+        else:
+            print(f"Success! Trustline to {code}:{asset_issuer_public_key} created!")
+
+    def add_asset_to_db(self, **options):
+        """
+        Add an asset to the database
+        """
+        asset_code = options.get("asset")
+        issuer = options.get("issuer_public")
+        distribution_seed = options.get("distribution_seed")
+        if not (issuer and asset_code):
+            print(f"\ninvalid value in options {options}")
+            return
+        for asset in Asset.objects.filter(issuer__isnull=False):
+            if asset.code == asset_code:
+                print(f"\nAsset {asset_code}:{issuer} already exists in the database, skipping...")
+                return
+        
+        print(f"\nAdding asset {asset_code}:{issuer} to database")
+        Asset.objects.create(
+            code=asset_code,
+            issuer=issuer,
+            distribution_seed=distribution_seed,
+            sep24_enabled=options.get("sep24_enabled") or True,
+            deposit_enabled=options.get("deposit_enabled") or True,
+            withdrawal_enabled=options.get("withdrawal_enabled") or True,
+            symbol="$"
```

Comment:
I think we should use whichever default value is defined in the DB, which I think is `False`. If we're trying to create a generic add-asset CLI command, we also. need to provide the ability to specify all fields, including symbol.

Reply:
Personally I don't think its worth the effort. Using `python manage.py shell` and adding the asset using the Django model is not terribly hard, especially with good example documentation and description of the fields.

Reply:
I like your middle-ground approach though, allowing devs to create the asset with the enabled SEPs but not specify fees or min/max limits.